### DISCUSSION TITLE
I made a little notepad with sVB!

### DISCUSSION THREAD

**Discussion by dynamicboy** [VBAndCs/sVB-Small-Visual-Basic#36] [Show and tell]
I make a little notepad with sVB.

LightThem:
![2023-12-05_031321](https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/b6323c42-34a5-4eca-aa28-f44058720f48)

DarkThem:
![2023-12-05_031330](https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/6bbaff7f-f0de-4c7a-96f7-9908f156afd6)

Zip File:
[FlashNote.zip](https://github.com/VBAndCs/sVB-Small-Visual-Basic/files/13550366/FlashNote.zip)

**Comment by VBAndCs** [comment]
Nice :)

________________________________
From: dynamicboy ***@***.***>
Sent: Monday, December 4, 2023 7:14 PM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] I made a little notepad with sVB! (Discussion #36)


I make a little notepad with sVB.

LightThem:
2023-12-05_031321.png (view on web)<https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/b6323c42-34a5-4eca-aa28-f44058720f48>

DarkThem:
2023-12-05_031330.png (view on web)<https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/6bbaff7f-f0de-4c7a-96f7-9908f156afd6>

Zip File:
FlashNote.zip<https://github.com/VBAndCs/sVB-Small-Visual-Basic/files/13550366/FlashNote.zip>

—
Reply to this email directly, view it on GitHub<https://github.com/VBAndCs/sVB-Small-Visual-Basic/discussions/36>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALQ5MVTC5LBWCXWD2AJEM5DYHYOJZAVCNFSM6AAAAABAGPN6RCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHEZDIMBQHE>.
You are receiving this because you are subscribed to this thread.

[https://s-install.avcdn.net/ipm/preview/icons/icon-envelope-tick-round-orange-animated-no-repeat-v1.gif]<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>  Virus-free.www.avast.com<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>

**Comment by VBAndCs** [comment]
I suggest to share such apps in this group:
https://www.facebook.com/groups/58053367137
________________________________
From: dynamicboy ***@***.***>
Sent: Monday, December 4, 2023 7:14 PM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] I made a little notepad with sVB! (Discussion #36)


I make a little notepad with sVB.

LightThem:
2023-12-05_031321.png (view on web)<https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/b6323c42-34a5-4eca-aa28-f44058720f48>

DarkThem:
2023-12-05_031330.png (view on web)<https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/6bbaff7f-f0de-4c7a-96f7-9908f156afd6>

Zip File:
FlashNote.zip<https://github.com/VBAndCs/sVB-Small-Visual-Basic/files/13550366/FlashNote.zip>

**Comment by VBAndCs** [comment]
I just published the first book of the "Small Visual Basic Kid Programmer" series on Amazon: https://www.amazon.com/dp/B0CVK1BMC9
________________________________
From: dynamicboy ***@***.***>
Sent: Monday, December 4, 2023 7:14 PM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] I made a little notepad with sVB! (Discussion #36)


I make a little notepad with sVB.

LightThem:
2023-12-05_031321.png (view on web)<https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/b6323c42-34a5-4eca-aa28-f44058720f48>

DarkThem:
2023-12-05_031330.png (view on web)<https://github.com/VBAndCs/sVB-Small-Visual-Basic/assets/7421525/6bbaff7f-f0de-4c7a-96f7-9908f156afd6>

Zip File:
FlashNote.zip<https://github.com/VBAndCs/sVB-Small-Visual-Basic/files/13550366/FlashNote.zip>
